### ISSUE TITLE
chore: WASM head doesn't build in the current state

### ISSUE THREAD

**Issue by benventive** [nventive/UnoApplicationTemplate#58]
## Expected Behavior
WASM head should build and the pipeline should be updated accordingly

## Actual Behavior
WASM head doesn't build

## Steps to Reproduce the Problem

1. Open the solution
2. Set the WASM project as the startup project
3. Build

## Specifications

- Commit: b59d7e819b4074a903b2dcc7ce5f0622a1f07100

**Comment by ExiledNarwal28** [comment]
So, I was able to do some things and I think I found the problem.

- The Wasm project cannot find `App.xaml`, which is in the UWP project. I weirdly was not able to simply reference the project.
- Chinook is missing in the Wasm project

* A perfect solution would be to move all views, assets, `App.xaml` and `Shell.xaml` to `UnoApplicationTemplate.Shared.Views`. Device-specific project (UWP, Droid, iOS, Wasm, ...) should only contain device-specific files, such as configurations and views overrides. I tried to do this, but completely broke Droid and iOS. I can look further into it or leave this to someone with more Uno experience than me!
